### PR TITLE
fix: url

### PR DESCRIPTION
**Pull request by harish-sethuraman**
Fixes https://github.com/reactjs/react.dev/issues/5868

### CODE REVIEW

**Comment by harish-sethuraman** [comment]
We have setup redirect in `vercel.json`?

**Comment by gaearon** [comment]
Yes but I'd like the links to be direct because this is clearer.

**Comment by harish-sethuraman** [comment]
Removed in redirect as well. Is that fine?

File: vercel.json [hunk 1]
```diff
@@ -139,11 +139,6 @@
       "destination": "/reference/react-dom/components/common#dangerously-setting-the-inner-html",
       "permanent": false
     },
-    {
```
Comment (gaearon) [inline vercel.json:142]: We should keep the redirect. We'll keep using it from React error messages.

**Pull request merged**